### ISSUE TITLE
Ubuntu 20.04 no longer supported?

### ISSUE THREAD

**Issue by IzzySoft** [ungoogled-software/ungoogled-chromium-binaries#444]
It looks like starting with 114, Ubuntu 20.04 and derivatives (like Mint 20) are no longer supported. Is there any chance to get working binaries for them again? After all, they are LTS releases supported until 2025. Would be great to have them supported by ungoogled chromium until then at least. Thanks in advance!

**Comment by clickot** [comment]
hi @IzzySoft  unfortunately i can't produce a build that works with ubuntu 20.04. For the reason see the later comments on https://github.com/ungoogled-software/ungoogled-chromium/issues/2001#issuecomment-1597463689 
The main point is that AppImages are not entirely independend from the operating system they run on; at least they depend on the glic of the OS (and maybe more). Since 114.xx chromium did not compile anymore with the older glibc version that is used in ubuntu 20.04 or debian bullseye, so i had to switch to bookworm to compile, what lead to an appimage that needs a higher glic version that ubuntu 20.04 has.
I recommend to use a flatpak version of ungoogled-chromium. Flatpaks are entirely independend from the underliying OS. See https://flathub.org/apps/com.github.Eloston.UngoogledChromium

**Comment by IzzySoft** [comment]
Thanks for the details! Flatpak is not an option for me, though. But hopefully I find some time during my "winter break" (the last 2 weeks of December) to upgrade my system, which should solve the issue for me then as well. Sad there's no such thing like a stable rolling LTS :see_no_evil:

**Comment by PF4Public** [comment]
> to upgrade my system

That would be a better solution

>stable rolling LTS

Gentoo?

Jokes aside, it is generally a good idea to update your distribution periodically without relying on a LTS terms. Another thing to consider is that we are moving this project voluntarily and oftentimes cannot spare more time just to additionally maintain LTS support.

**Comment by IzzySoft** [comment]
> That would be a better solution

Definitely.

> Gentoo?

"I'm using…"

> Jokes aside

OK, I didn't complete mine – not after your edit of yours :see_no_evil:

And you are mostly right with the rest. Thing is, in some environments there is a reason for the "L" in "LTS". On the machine in question I have a rather complex setup. So security updates are of course applied in a timely manner, and usually go through "seamlessly", But dist-upgrades are a different thing, they require a lot of planning and manual work, so doing them more frequent than every 3 years or so would be quite an overload. I rarely stick with an LTS "full term", but I have to find a slot where I can handle the load. Unfortunately there are also such things as `$dayjob` and (in my case) volunteer work (eating 3-4h each day).

TL;DR: It's not possible to make all the people happy all the time, for neither of us. So we can see what's doable, and make the best out of it. Thanks for your work, and for your understanding!

**Comment by clickot** [comment]
Hello @IzzySoft i managed to build the latest ungoogled-chromium version on a debian bullseye image, so the result does run on ubuntu 20.04 again, see https://github.com/ungoogled-software/ungoogled-chromium-portablelinux/releases/tag/119.0.6045.123-1

**Comment by Samueru-sama** [comment]
bump

This nightmare is avoided by bundling glibc, like I do [here with Cromite](https://github.com/pkgforge-dev/Cromite-AppImage): 

Works in 16.04. (as well as non glibc distros). 

<img width="1426" height="873" alt="Image" src="https://github.com/user-attachments/assets/40094588-cb4a-4190-81a9-efc699b92aa6" />
